db/seeds/zip3_dtod.py

```diff
@@ -0,0 +1,16 @@
+
+for i in range(len(zip3)):
+  # Zero-pad on the left to ensure that the ZIP3 is actually 3 digits
+  # Add two spaces on the right to ensure that the field is 5 characters, because that's how DTOD rolls
```

Comment:
D:

Reply:
I was told in no uncertain terms that DTOD does not accept 3 digit zip codes. When I noticed that 3 digit zip codes worked on DTOD's website, I figured there must be a way to make it work in batch mode... and there was.